kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
How did this file get here? It doesn't look generated.

Reply:
That one is hard coded! I dropped it there for anyone wanting to use the bundle.yaml as their kustomize base.